pkg/networkutils/network.go

```diff
@@ -115,10 +122,19 @@ type iptablesIface interface {
 	NewChain(table, chain string) error
 }
 
+type snatType uint32
+
+const (
+	sequentialSNAT snatType = 0
+	randomHashSNAT snatType = 1
+	randomPRNGSNAT snatType = 2
```

Comment:
The idiomatic way to define this is to use `iota`. (See https://golang.org/ref/spec#Iota)